src/platform/user/profile/vet360/components/PhoneField/PhoneEditModal.jsx

```diff
@@ -8,13 +8,17 @@ import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
 
 import Vet360EditModal from '../base/Vet360EditModal';
 
-import { getEnrollmentStatus as getEnrollmentStatusAction } from 'applications/hca/actions';
```

Comment:
This action was used in mapDispatchToProps but the prop was never actually used in the component. It was passed down to other components via `...this.props` spreads, but also not used by those.